### PR TITLE
Verify GPU Items Appears In Resources Page

### PR DESCRIPTION
**Pull request by risusing**

### CODE REVIEW

**Review by bdattoma — APPROVED** [review]
lgtm. Please test it on Jenkins

**Comment by risusing** [comment]
Jenkins Job Run : https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/543/
Result : Passed.

File: tests/Tests/400__ods_dashboard/400__ods_dashboard.robot [hunk 1]
```diff
@@ -94,6 +94,12 @@ Verify Documentation Link HTTP Status Code
     ${links}=  Get RHODS Documentation Links From Dashboard
     Check External Links Status     links=${links}
 
+Verify GPU Items Appears In Resources Page
+    [Tags]  ODS-1226
```
Comment (tarukumar) [inline tests/Tests/400__ods_dashboard/400__ods_dashboard.robot:98]: How about **Search And Verify GPU Items Appears In Resources Page** as keyword name .

**Review by tarukumar — APPROVED** [review]
LGTM.

I have added one small suggestion

File: tests/Tests/400__ods_dashboard/400__ods_dashboard.robot (hunk 1 above)
Comment (risusing) [reply tests/Tests/400__ods_dashboard/400__ods_dashboard.robot:98]: Yes that sounds more specific.

**Review by tarukumar — DISMISSED** [review]
LGTM

File: tests/Tests/400__ods_dashboard/400__ods_dashboard.robot [hunk 2]
```diff
@@ -94,6 +94,12 @@ Verify Documentation Link HTTP Status Code
     ${links}=  Get RHODS Documentation Links From Dashboard
     Check External Links Status     links=${links}
 
+Search and Verify GPU Items Appears In Resources Page
+    [Tags]  ODS-1226
+    Click Link  Resources
+    Sleep   5
+    Input Text  xpath://input[@class="pf-c-search-input__text-input"]   GPU
```
Comment (bdattoma) [inline tests/Tests/400__ods_dashboard/400__ods_dashboard.robot:101]: You could wrap up these 3 lines in a Keyword called `Search Item In Resource Section` which takes the search term as argument and put this keyword in the Resource file for the dashboard

**Review by bdattoma — APPROVED** [review]
lgtm

**Review by bdattoma — DISMISSED** [review]

**Review by milindw96 — DISMISSED** [review]

**Comment by milindw96** [comment]
hi @risusing  can u plz resolve the merge conflict (rebase it )

**Review by bdattoma — APPROVED** [review]

**Review by milindw96 — APPROVED** [review]

**Pull request merged**